src/djangosaml2_spid/tests.py

```diff
@@ -166,37 +166,37 @@ def test_spid_public_cert(self):
             'cert_file': 'example/certificates/public.cert'
         }])
 
+#
+# class TestStaticFiles(TestCase):
 
-class TestStaticFiles(TestCase):
+    # def test_spid_logo(self):
+        # abs_path = finders.find('spid/logo.jpg')
+        # self.assertTrue(os.path.isfile(abs_path))
 
-    def test_spid_logo(self):
```

Comment:
Hey @brunato 
I just omitted them, feel free to put them again (even in a next release).
Now also the demo server is a quite real one, with collected statics, tls certs and things.